oca_event_badge/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 ACSONE SA/NV
+# Copyright 2018 Tecnativa - Cristina Martin R.
```

Comment:
2019

Reply:
Yes, I almost forget it. Change done. 👍🏻